templates/services_user_data.tpl

```diff
@@ -110,9 +110,10 @@ mkdir -p /etc/circleconfig/telegraf
 
 cat <<EOF > /etc/circleconfig/telegraf/telegraf.conf
 [global_tags]
-  service = "circleci"
-  type = "server"
+  service = "circleci-server"
+  app = "circleci"
   env = "${env}"
+  host = "$(echo $HOSTNAME)"
```

Comment:
since this telegraf configuration is only for `services` instance and that is only one instance (per environment), should we not use something like `circleci-${env}` or something similar instead of using the `$HOSTNAME`?

Reply:
Some thoughts:

- What if we want to scale the master/services server?
- Given the metrics are for the machine, `$HOSTNAME` makes the most sense as it will be the easiest way to locate the instance (given the hostname is something like `ip-xx-xx-xx-xx`).

Reply:
It definitely makes sense to use the `$HOSTNAME` if we had a valid hostname. Unfortunately, the CircleCI services instance cannot be horizontally scaled and there is nothing on CircleCI's roadmap to indicate otherwise. So there is ever going to be only one server live per environment unfortunately. Hence the reasoning for having a simpler name that could easily be picked from the tags on Datadog rather than a random private dns name